@ecomplus/storefront-components/src/html/TheAccount.html

```diff
@@ -88,7 +88,7 @@ <h4>{{ i19addresses }}</h4>
         enter-active-class="animated fadeInRight slow"
         leave-active-class="animated fadeOutRight position-absolute"
       >
-        <div v-if="currentTab === 'subscriptions'">
+        <div v-if="currentTab === 'subscriptions' && subscriptions.length">
```

Comment:
Com o array `navTabs` e o push asíncrono nele você não precisa disso aqui porque antes de ele ter length 4 o usuário nunca chega a setar `currentTab === 'subscriptions'`, certo?

```suggestion
        <div v-if="currentTab === 'subscriptions'">
```